shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/compiler/ProgramNode.java

```diff
@@ -118,7 +118,7 @@ public void render(RootBuilder builder) {
   }
 
   public int timeout() {
-    AtomicInteger timeout = new AtomicInteger(20);
+    AtomicInteger timeout = new AtomicInteger(20 * 600);
```

Comment:
Am I misreading this, or should this be `20 * 60`?

Reply:
:man_facepalming: Fixed.